### ISSUE TITLE
Error if not element provided

### ISSUE THREAD

**Issue by JamesJGoodwin** [atomiks/tippyjs#114]
Hello. I've met kind of issue with tippy. I have an `app.js` file which is connected on every page in my project. I've decided to initialize tooltips I need in `app.js`(to not to duplicate the code on each page):

```javascript
tippy(document.querySelector('.some-block'));
```
But after I did so, I keep recieving an error on pages where `.some-block` not exists:
>Cannot read property 'constructor' of null

The issue of such behavior is that the error disables some JavaScript listeners(such as `$(window).on('scroll', function(){});`) which is breaking usability of the whole page. I've coded a small hack to avoid this kind of behavior:
```javascript
if($('*').is('.some-block')) {
    tippy(document.querySelector('.some-block'));
}
```

Could you, please, catch situations like this and throw a warning in the console instead of error?

Much thanks in advance.

**Comment by atomiks** [comment]
Good call, I think wrapping the final return in try/catch should solve that

**Comment by atomiks** [comment]
Fixed in `1.4.0`
